app/design/adminhtml/Magento/backend/Magento_Sales/web/css/source/module/order/_payment-shipping.less

```diff
@@ -102,8 +106,9 @@
     + .order-payment-currency {
         margin-top: @indent__s;
     }
+    
     .admin__table-secondary {
         margin-top: @indent__s;
         &:extend(.abs-admin__table-secondary-edit-order all);
     }
-}
+}
```

Comment:
Could you add empty line at the end?

Reply:
oops i will revert that

Reply:
updated and rebase code mainline to my PR